README.md

```diff
@@ -38,15 +38,38 @@ python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 
 ## Currently supported
 
-Operations - Contraction Tensor
-Data Types - FP32 , FP64
+### Operations - Contraction Tensor
```

Comment:
Does "Contractor Tensor" need to be capitalized (i.e. is it a formal name)? All titles should be in sentence case.